hyperspy/misc/utils.py

```diff
@@ -1226,3 +1232,19 @@ def is_hyperspy_signal(input_object):
     """
     from hyperspy.signals import BaseSignal
     return isinstance(input_object,BaseSignal)
+
+
+def is_binned(signal, axis=-1):
+    """Backwards compatibility check utility for is_binned attribute.
+    
+    Can be removed in v2.0.
+    """
+    if signal.metadata.has_item('Signal.binned'):
+        warnings.warn('Use of the `binned` attribute in metadata is going to '
```

Comment:
An option could be to use a `DeprecationWarning` instead of `VisibleDeprecationWarning`, but I can also remove it altogether.

Reply:
What do you prefer @ericpre?

Reply:
I think the warning in DTB is good enough and is acting at the right place and I would remove this warning. Since the binned information is set correctly when loading files, this is only when an user sets it in the metadata that it would this situation would occur, which case, a warning would already have been raised.

Reply:
good point